site/en/docs/android/custom-tabs/guide-partial-custom-tabs/index.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: "layouts/doc-post.njk"
+title: Resizing with partial Custom Tabs
+seoTitle: "Guide: resize a Custom Tab with Partial Custom Tabs"
+date: 2023-04-21
+description: Learn how to you use partial Custom Tabs for letting your users interact with your app while viewing web content. 
+authors:
+  - sebastianbenz
+---
+
+By default, Custom Tabs launch as a full-window activity. Starting in Chrome 107, you can use partial Custom Tabs to specify a different launch height in portrait mode such that users can interact with your app while viewing web content. Users can expand the Custom Tab to full-screen by dragging the toolbar handle up and restoring the initial launch height by dragging the handle down.
+
+{% Img src="image/6AZNJBRnkpQUWTKPzig99lQY8jT2/iMTvWNK7aEqqxHEHEXuv.png", alt="Example Partial Tab", width="320", height="640", class="screenshot screenshot--filled" %}
+
+To turn a Custom Tab into a partial Custom Tab, define the initial launch height in pixels by calling the  CustomTabBuilder’s [setInitialActivityHeightPx](https://developer.android.com/reference/androidx/browser/customtabs/CustomTabsIntent.Builder#setInitialActivityHeightPx(int)) method. By default, the partial Custom Tab is resizable, but you can pass [ACTIVITY\_HEIGHT\_FIXED](https://developer.android.com/reference/androidx/browser/customtabs/CustomTabsIntent#ACTIVITY_HEIGHT_FIXED()) to disable this behavior:
+
+```java
+new CustomTabsBuilder().setInitialActivityHeightPx(
+    400,
+    ACTIVITY_HEIGHT_FIXED
+);
+```
+
+{% Aside 'gotchas' %}
+The minimum partial custom tab height is 50% of the screen height, if the screen height is set to a value less than 50% of the screen height, Chrome automatically adjusts the Custom Tab to 50% of the screen height. 
+{% endAside %}
+
+Furthermore, you need to either:
+
+1. [start a new browser session via a `CustomTabsServiceConnection`](/docs/android/custom-tabs/integration-guide/#connect-to-the-custom-tabs-service) and pass it to the Custom Tabs intent or
+
+2. start the Custom Tab activity via [`startActivityForResult`](https://developer.android.com/reference/android/app/Activity#startActivityForResult(android.content.Intent,%20int)).
+
+Combine both approaches if you want to guarantee a fast startup in case the service connection has not been established yet. 
+
+{% Aside 'gotchas' %}
+* Specifying the initial activity height will not have an effect if the default browser does not support resizing the Custom Tab. In this case, the intent extra will be ignored and the Custom Tab will span the complete display height.
+* Custom Tabs will inherit the host app's color scheme for the user interface properties above. You will be responsible for ensuring visual consistency for these properties before launching a Custom Tab. This means the `CustomTabColorScheme.navigationBarColor` and `CustomTabColorScheme.navigationBarDividerColor` properties do not work when building an intent for a partial Custom Tab.
```

Comment:
> the user interface properties above

above what?

Reply:
For clarity, i'd reorder this section a tiny bit

`CustomTabColorScheme.navigationBarColor` and `CustomTabColorScheme.navigationBarDividerColor` properties do not work when building an intent for a partial Custom Tab. This is because they inherit the host app's color scheme for the user interface properties above. You will be responsible for ensuring visual consistency for these properties before launching a Custom Tab.

Reply:
good suggestion! thanks!